e2e/types.go

```diff
@@ -142,9 +142,7 @@ type Config struct {
 	// VMConfigMutator is a function which mutates the base VMSS model according to the scenario's requirements
 	VMConfigMutator func(*armcompute.VirtualMachineScaleSet)
 
-	// LiveVMValidators is a slice of LiveVMValidator objects for performing any live VM validation
-	// specific to the scenario that isn't covered in the set of common validators run with all scenarios
-	LiveVMValidators []*LiveVMValidator
+	Validator func(ctx context.Context, s *Scenario)
```

Comment:
This is the core difference

Reply:
could you explain the main reason for this change ?